src/main/java/com/target/studentinfo/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,25 @@
+package com.target.studentinfo.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(BadRequestException.class)
+    public ResponseEntity<String> handleRequestValidationException(BadRequestException ex) {
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(ex.getMessage());
+    }
+
+    @ExceptionHandler(NotFoundException.class)
+    public ResponseEntity<String> handleNoHandlerFoundException(NotFoundException ex) {
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(ex.getMessage());
```

Comment:
When you throw exception, you're setting message, errorCode and httpStatus but while handling, you're only using ex.getMessage(). We can pass our custom errorCode here. We'll discuss.

Reply:
```
{
    "errorCode": "MISSING_FIELD",
    "errorMessage": "some of the required fields misisng"
}
```
Try and achieve this as exception response with status code 400.